onnx_tf/frontend.py

```diff
@@ -201,6 +202,38 @@ def tensorflow_graph_to_onnx_graph(cls, graph_def, output, opset=0, name="graph"
                       output_proto,
                       consts_proto)
 
+  @classmethod
+  def tensorflow_graph_to_onnx_model(cls,
+                                     graph_def,
+                                     output,
+                                     opset=0,
+                                     producer_name="onnx-tensorflow",
+                                     graph_name="graph"):
+    """Converts a Tensorflow Graph Proto to an ONNX model
+
+    This function converts a Tensorflow Graph proto to an equivalent
+    representation of ONNX model.
+
+    :param graph_def: Tensorflow Graph Proto object.
+    :param output: A Tensorflow NodeDef object specifying which node
+      to be taken as output of the ONNX graph.
+    :param opset: Opset version of the operator set.
+      Default 0 means using latest version.
+    :param producer_name: The name of the producer.
```

Comment:
A little question, what does producer do?

Reply:
Its the name of framework/tool that generated this model.

Reply:
I see. User can change this to any name they want.

Reply:
Yes, I was hoping to allow users to specify this field if they use our tool as an API and they want to give credit to themselves. I don't think we should allow user to modify it if user generates anything using our command line tool.